tests/experimental/pydantic/test_conversion.py

```diff
@@ -55,6 +57,68 @@ class UserType:
     assert user.password == "abc"
 
 
+def test_cannot_convert_pydantic_type_to_strawberry_missing_field():
+    class User(BaseModel):
+        age: int
+
+    with pytest.raises(
+        AutoFieldsNotInBaseModelError,
+        match=re.escape(
+            "UserType defines ['password'] with strawberry.auto."
+            " Field(s) not present in User BaseModel."
+        ),
+    ):
+
+        @strawberry.experimental.pydantic.type(User)
+        class UserType:
+            age: strawberry.auto
+            password: strawberry.auto
+
+
+def test_cannot_convert_pydantic_type_to_strawberry_property_auto():
+    # auto inferring type of a property is not supported
+
+    class User(BaseModel):
+        age: int
+
+        @property
+        def password(self) -> str:
+            return "hunter2"
+
+    with pytest.raises(
+        AutoFieldsNotInBaseModelError,
+        match=re.escape(
+            "UserType defines ['password'] with strawberry.auto."
+            " Field(s) not present in User BaseModel."
+        ),
+    ):
+
+        @strawberry.experimental.pydantic.type(User)
+        class UserType:
+            age: strawberry.auto
+            password: strawberry.auto
+
+
+def test_can_convert_pydantic_type_to_strawberry_property():
+    class User(BaseModel):
+        age: int
+
+        @property
+        def password(self) -> str:
+            return "hunter2"
+
+    @strawberry.experimental.pydantic.type(User)
+    class UserType:
+        age: strawberry.auto
+        password: str
+
+    origin_user = User(age=1)
+    user = UserType.from_pydantic(origin_user)
+
+    assert user.age == 1
+    assert user.password == "hunter2"
```

Comment:
making sure that this previous capability remains

Reply:
💯